test/e2e/upgrade/dns/dns.go

```diff
@@ -0,0 +1,129 @@
+package dns
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"github.com/onsi/ginkgo"
```

Comment:
We follow a conventional pkg ordering pattern of:
* single-name pkg alphabetically ordered
* blank line
* github pkgs alphabetically ordered
* blank line
* k8s pkgs alphabetically ordered
* blank line
* other pkgs in alphabetical order

Reply:
Thanks! I re-ordered them based on this. Is there a place where I can see all the team conventions?

Reply:
Mostly just what `go fmt` and `goimports` do.  There are also https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#api and https://go.dev/doc/effective_go#formatting